### DISCUSSION TITLE
How to pass BigInt as an argument type to the query?

### DISCUSSION THREAD

**Discussion by winsoroaks** [0no-co/gql.tada#103] [Q&A] [answered]
Hi all. I am trying to pass an argument that is of BigInt type as a condition to my query but was not successful.
```ts
query MyQuery($number: BigInt!) {
  counts(where: { number: $number }) {
```
im getting 
```
TypeError: Do not know how to serialize a BigInt
    at JSON.stringify (<anonymous>)
    at createRequestBody (index.js:381:32)
```
i then tweaked my schema for the number to be of type String and make the number to be of type String! in the query. i then ran into 
```
Error: Variable "$number" of type "String!" used in position expecting type "BigInt".
```
Please let me know if this is the wrong place to ask the question. Thank you.

**Comment by JoviDeCroock** [comment, marked as answer]
The error does not come from this library but from createRequestBody which is probably from the library that sends your gql requests
